adversarial-thinking.adoc

```diff
@@ -104,16 +104,16 @@ completely isolating a node from the honest network can be helpful in
 the execution of other attacks.
 ____
 
-This type of attack is called a _Sybil attack_, which means that a
-single entity controls multiple nodes in a network to make it appear
+This type of attack is called _Sybil attack_, and it occurs whenever a
+single entity controls multiple nodes in a network and uses them to appear
 as multiple entities.
 
 As the quote also mentions, the Sybil attack is not effective on the
-Bitcoin network because there is no voting with nodes or entities, but
-<<one-cpu-one-vote,with computing power>>. However, this flat
+Bitcoin network because there is no voting through nodes or other numerable entities, but rather
+<<one-cpu-one-vote,through computing power>>. Nonetheless, this flat
```

Comment:
This is probably a good exception to the "techical xrefs" rule. We are referring to a key paragraph somewhere else in the book, which doesn't have a number.

Reply:
@kallerosenbaum I gave this a thought and realized that it's easier if I review the content first and then, at a later stage, do the typesetting, including review of all internal and external references. What do you think about this option?

Also, are we erasing all the external link from the text at some point? I wouldn't leave things like "watch the video here" in a printed book

Reply:
> What do you think about this option?

Sounds good to me! I'll stop commenting on such issues from now on then. But I don't mind if you change xrefs as you go either. We'll probably go over all the links and xrefs in the end regardless of what you choose here.

> Also, are we erasing all the external link from the text at some point? I wouldn't leave things like "watch the video here" in a printed book

I just had an idea: Let's add small QR codes for the external links in the margin of the book and also have a list of the full textual URLs as an appendix. This could most likely be done automatically using some script magic. For ebook/PDF, it still makes sense to keep the links in the text. What do you think about this?

Reply:
Love the QR codes and appendix idea! This way we won't have to sacrifice the multi-layered complexity of the information we are covering

Reply:
Great! I'll see if I can fix the QR codes, then.

Reply:
Created issue https://github.com/bitcoin-dev-philosophy/btcphilosophy/issues/79 for QR codes